### PR TITLE
 ISSD-2097 [Marketplace] Fix Email templates to Gmail pattern

### PR DESCRIPTION
**Pull request by welitonpe**
**[ISSD-2097](https://liferay.atlassian.net/browse/ISSD-2097)**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/publisher-account-request/en-US.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+									<a href="#" style="color: #0B5FFF;text-decoration: none;">
+										Liferay, Inc.
+									</a>
+								</span>
+							</p>
+						</div>
+
+						<div style="margin-left:auto; margin-right:auto; width:fit-content;">
+							<img alt="Liferay Logo" src="[$PORTAL_URL$]/documents/d/[$GROUP_KEY$]/liferay_logo-png" style="margin-top: 32px; width: 180px; height: 56.9px; margin-left:auto; margin-right:auto; margin-bottom: 32px;" />
+						</div>
```
Comment (juliocezar96) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/publisher-account-request/en-US.html:72]:
```suggestion
							<img alt="Liferay Logo" src="[$PORTAL_URL$]/documents/d/[$GROUP_KEY$]/liferay_logo-png" style="height: 56.9px; margin-bottom: 32px;  margin-top: 32px; margin-left:auto; margin-right:auto; width: 180px; " />
						</div>
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/cloud-next-steps-template/en-US.html [hunk 2]
```diff
@@ -1,54 +1,51 @@
...
+
+<head>
+	<title>
+		Cloud App Template
+	</title>
+</head>
+
+<body>
+	<div style="background-image: url('[$PORTAL_URL$]/documents/d/[$GROUP_KEY$]/backgroud-png');background-size: cover; background-position: left; display: flex; font-family: 'Source Sans Pro', sans-serif;">
+		<div style="background-color: #fff; border-radius: 16px; border: 1px solid #6b6c7e; margin-bottom: 4rem; margin-top: 4rem;margin-left: auto; margin-right: auto; padding: 3rem; width: 925px;">
```
Comment (juliocezar96) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/cloud-next-steps-template/en-US.html:13]:
```suggestion
		<div style="background-color: #fff; border: 1px solid #6b6c7e; border-radius: 16px;  margin-bottom: 4rem;  margin-left: auto; margin-right: auto;  margin-top: 4rem; padding: 3rem; width: 925px;">
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/invite-member-template/en-US.html [hunk 3]
```diff
@@ -1,23 +1,24 @@
...
+	</title>
+</head>
 
-					<h1>Marketplace</h1>
-				</div>
-
-				<div style="background-color: #6b6c7e; height: 1px; margin-top: 1.813rem; width: 42rem;">
+<body>
+	<div style="align-items: center; background-image: url('[$PORTAL_URL$]/documents/d/[$GROUP_KEY$]/backgroud-png'); background-position: left; display: flex; flex-direction: column; justify-content: center;">
+		<div style="background-color: #fff; border-radius: 4px; border: 1px solid #6b6c7e; height: 50rem; margin-top: 6.25rem; padding: 3rem; width: 43.375rem;">
```
Comment (juliocezar96) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/invite-member-template/en-US.html:15]:
```suggestion
		<div style="background-color: #fff; border: 1px solid #6b6c7e;  border-radius: 4px; height: 50rem; margin-top: 6.25rem; padding: 3rem; width: 43.375rem;">
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/publisher-account-request/en-US.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+				<div style="margin-bottom: 40px;">
+					<p>
+						A new request has been submitted to the Marketplace for a publisher account. Please review the
+						submission and create the publisher account. Invite the user to the account as per existing
+						Marketplace functionality.
+					</p>
+				</div>
+
+				<a href="#" style="background-color: #0b5fff; border-radius: 8px; border: none; color: #fff; height: 2.5rem; margin-top: 2.5rem; padding: 0.5rem 1.5rem; text-decoration: none; width: 10.938rem;">
```
Comment (juliocezar96) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/publisher-account-request/en-US.html:41]:
```suggestion
				<a href="#" style="background-color: #0b5fff; border: none; border-radius: 8px; color: #fff; height: 2.5rem; margin-top: 2.5rem; padding: 0.5rem 1.5rem; text-decoration: none; width: 10.938rem;">
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/publisher-account-request/en-US.html (hunk 1 above)
Comment (juliocezar96) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/publisher-account-request/en-US.html:72]:
```suggestion
							<img alt="Liferay Logo" src="[$PORTAL_URL$]/documents/d/[$GROUP_KEY$]/liferay_logo-png" style=" height: 56.9px; margin-bottom: 32px; margin-left:auto; margin-right:auto; margin-top: 32px; width: 180px;" />
						</div>
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/publisher-account-request/en-US.html [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+								1400 Montefino Ave
+							</div>
+
+							<div style="color:#6B6C7E; margin-top: 4px;">
+								Diamond Bar, CA 91765
+							</div>
+						</div>
+					</div>
+
+					<div style="font-size: 14px; text-align:center; ">
```
Comment (juliocezar96) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/publisher-account-request/en-US.html:89]:
```suggestion
					<div style="font-size:14px; text-align:center;">
```

**Review by juliocezar96 — CHANGES_REQUESTED** [review]
Hey, Nice job! But, I have some suggestions. 🧑‍💻 👀

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Comment by kevenleone** [comment]
resent:T https://github.com/brianchandotcom/liferay-portal/pull/145261

**Pull request closed without merging**